st2common/st2common/util/action_db.py

```diff
@@ -195,6 +196,11 @@ def update_liveaction_status(status=None, result=None, context=None, end_timesta
     old_status = liveaction_db.status
     liveaction_db.status = status
 
+    liveaction_db.log.append({
```

Comment:
We already have mongo specific code. I am divided honestly. I don't see the need here to use mongo specific atomic updates but I might be missing something. cc: @manasdk, @Kami